neuralop/models/gino.py

```diff
@@ -191,16 +223,39 @@ def __init__(
         if self.fno_norm == "ada_in":
             if fno_ada_in_features is not None:
                 self.adain_pos_embed = SinusoidalEmbedding(in_channels=fno_ada_in_dim,
-                                                           num_frequencies=fno_ada_in_features,
-                                                           embedding_type='transformer',
-                                                           max_positions=gno_embed_max_positions)
+                                                           num_frequencies=fno_ada_in_features, 
+                                                           max_positions=10000,
+                                                           embedding_type='transformer')
```

Comment:
I am not sure should we set the embedding_type to be gno_pos_embed_type

Reply:
Good point, changed